### DISCUSSION TITLE
TonalJS REST api

### DISCUSSION THREAD

**Discussion by martijnmichel** [tonaljs/tonal#278] [Show and tell]
Ive created a small REST api for tonaljs and deployed it on ZEIT. 

https://tonaljs-rest-api.now.sh/

available urls for now are:

CHORD
list all chords > /chord
get one chord > /chord/:chordalias
get transposed > /chord/:root/:chordalias

SCALE
list all scales > /scale
get one scale > /scale/:scalename
get transposed > /chord/:root/:scalename

INTERVAL
list an interval > /interval/:name
tranposed > /interval/:root/:name

Lets see what we can do with it ;p

**Comment by danigb** [comment]
Nice! 👏

**Comment by rowild** [comment]
Thank you! Awesome! Bookmarked!
Would be great, if the root age could have a bit of a design and explanation.

**Comment by martijnmichel** [comment]
What do you mean?

btw: https://github.com/martijnmichel/tonaljs-rest-api

**Comment by rowild** [comment]
I just thought that, when calling https://tonaljs-rest-api.now.sh/ directly there should be sth else but an error message... but its just icing on the cake! :-)
